.devcontainer/Dockerfile

```
@@ -47,6 +47,14 @@ RUN export PORTER_HOME=/home/vscode/.porter && curl -L https://cdn.porter.sh/lat
 
 ENV PATH /home/vscode/.porter/:$PATH
 
+# Install Azure Function Core Tools
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg \
+    && sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg \
+    && sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/debian/$(lsb_release -rs | cut -d'.' -f 1)/prod $(lsb_release -cs) main" > /etc/apt/sources.list.d/dotnetdev.list' \
+    && sudo apt-get update \
+    && sudo apt-get update \
```

I think we only need one apt-get update
```suggestion
    && sudo apt-get update \
```